### DISCUSSION TITLE
How to fix "version `GLIBC_2.27' not found" error?

### DISCUSSION THREAD

**Discussion by theprinceraj** [vercel/community#5036] [Help]
The code of my project is at: [https://github.com/theprinceraj/Profile-Cards-API](https://github.com/theprinceraj/Profile-Cards-API) 

The error as shown in the vercel logs when I try to access the website is:
**Error: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by /var/task/node_modules/skia-canvas/lib/v6/index.node)**

I have tried to solve it using the fix provided in the discussion at: [https://github.com/vercel/community/discussions/2867#discussioncomment-6254610](https://github.com/vercel/community/discussions/2867#discussioncomment-6254610)
The possible fix provided in the above discussion is changing the node version from 18.x to 16.x within vercel . I did the same, and then redeployed my main branch but it still gives the same error.
Also, there isn't a problem with the code as its running completely fine when I run it on my computer locally.

I think this detail might be helpful too so I will mention it: I earlier used @napi-rs/canvas to form images but I had switch to skia-canvas and canvas-constructor for some reason recently. Ever since, it has not been able to be run properly when deployed.

**Comment by TooTallNate** [comment]
Try setting Node version to `20.x` instead.

**Reply by theprinceraj** [reply]
@TooTallNate I have already tried it but that does not fix this issue either.
